docs/dsl/circle.rst

```diff
@@ -14,6 +14,26 @@ radius
 
   radius of the circle. Supports :doc:`/units`.
 
+arc_start
+  default: ``0``
+
+  angle on the circle to start an arc
+
+arc_end
+  default: ``2 * Math::PI``
+
+  angle on the circle to end an arc
+
+arc_ccw
```

Comment:
Let's call it `arc_direction` and have it default to `:clockwise` and then change if it's `:counterclockwise`.